proto/metrics/metrics.proto

```
@@ -0,0 +1,53 @@
+syntax = "proto3";
+
+package metrics;
+
+import "validate/validate.proto";
```

Buf (1.47.2)

import "validate/validate.proto": file does not exist

(COMPILE)